server/lib/configuration.js

```diff
@@ -283,6 +283,31 @@ var conf = module.exports = convict({
       doc: 'Location of "report-uri"',
       default: '/_/csp-violation',
     }
+  },
+  marketing_email: {
```

Comment:
We might as well take the `marketing_email` section out of this PR and add it to #2452.